gui/widget.cpp

```diff
@@ -292,11 +292,12 @@ void Widget::read(Common::String str) {
 #pragma mark -
 
 StaticTextWidget::StaticTextWidget(GuiObject *boss, int x, int y, int w, int h, const Common::String &text, Graphics::TextAlign align, const char *tooltip, ThemeEngine::FontStyle font)
-	: Widget(boss, x, y, w, h, tooltip), _align(align) {
+	: Widget(boss, x, y, w, h, tooltip) {
 	setFlags(WIDGET_ENABLED);
 	_type = kStaticTextWidget;
 	_label = text;
 	_font = font;
+	_align = Graphics::convertTextAlignH(align, g_gui.useRTL() && _useRTL);
```

Comment:
use the new alignment abstraction. if manual align is passed (instead of start/end), the function will jump to the default switch case and return the same thing.